packages/yuudachi/src/functions/settings/getGuildSetting.ts

```diff
@@ -11,6 +11,8 @@ export enum SettingsKeys {
 	Locale = 'locale',
 	LogIgnoreChannels = 'log_ignore_channels',
 	AutomodIgnoreRoles = 'automod_ignore_roles',
+	GeneralLogChannelId = 'general_log_channel_id',
+	SponsorRoleId = 'sponsor_role_id',
```

Comment:
This change is not relevant to this PR.

Reply:
Agreed, reverted in [8781e25](https://github.com/Naval-Base/yuudachi/pull/1069/commits/8781e25a498f6169c2d121d1887139dbc28b0a64)